tests/conftest.py

```diff
@@ -33,20 +34,22 @@ def cfg_default():
     return cfg
 
 
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="session")
```

Have you tried this? If that works, it's simpler than adding `lru_cache`

Reply:
Yes, it doesn't work because apparently `monkeypatch` is function scoped and therefore cannot be combined with a session scoped fixture

Reply:
Oh wait, I tried adding `scope="session"` to the `ignore_existing_user_config` fixture, not this one. Adding it here would solve the problem of reloading the data, but then there is the risk of modifying. I can try using a mappingproxy instead of copying though

Reply:
As far as I understand, the output of the fixture is cached when a wider scope is set, and then this output is reused. I don't think that this output can be modified. I might be wrong though..

Reply:
I forgot that CFG is not a plain dict but a custom object, so the mappingproxy thing doesn't work. The output of the fixture is an `esmvalcore.config.Config` object, which can be modified, so reusing it is not safe because then one test could break another test depending on the order in which you run them.

Reply:
Good point, agreed.